### PR TITLE
Show current user + Side Nav

### PR DESCRIPTION
**Pull request by D0nPiano**
Since in @tiltec  's request the buttons weren't shown in mobile, I thought of doing a little menu for the three buttons.

Well, and now I overdid it, and now it became a sidenav. :D

### CODE REVIEW

File: client/app/locales/locale-de.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "REDIRECT": "Umleitung zu deiner Gruppe..."
   },
   "TOPBAR": {
-    "LOGOUT": "Ausloggen"
+    "LOGOUT": "Ausloggen",
+    "GREETING": "Moin, {{username}}!"
```
Comment (tiltec) [inline client/app/locales/locale-de.json]: We need to enter this translation in transifex later, otherwise it gets lost on update.

**Comment by tiltec** [comment]
Looks good! We might need something like this in future anyway.
Just add the missing test for toggleRight (and perhaps move the function to the controller), then we can merge it :)

File: client/app/locales/locale-de.json (hunk 1 above)
Comment (D0nPiano) [inline client/app/locales/locale-de.json]: We always keep the english version though, right? So it just doesn't make sense to translate into other languages?

File: client/app/components/_topbar/topbar.controller.js [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+  constructor(Authentication, $mdMedia, $scope, $mdSidenav) {
+    "ngInject";
+    $scope.screenIsSmall = !$mdMedia("gt-sm");
+    $scope.$watch(() => {
+      return $mdMedia("gt-sm");
+    }, (big) => {
+      $scope.screenIsSmall = !big;
+    });
+
+    $scope.toggleRight = () => {
```
Comment (D0nPiano) [inline client/app/components/_topbar/topbar.controller.js]: Ups, nope. No clue why I used scope there... just changed it (:

File: client/app/locales/locale-de.json (hunk 1 above)
Comment (tiltec) [inline client/app/locales/locale-de.json]: yes, we use locale-en.json as source file for all translations. so the workflow is:
write key-value pair in `locale-en.json`, merge it to master, upload it to transifex, translate it, download the resulting `locale-**.json` files

Might put this in the README ;)

**Pull request merged**